playbooks/byo/config.yml

```diff
@@ -1,2 +1,14 @@
 ---
+- name: Verify Requirements
+  # REVIEW: what's the proper group to use: OSEv3, g_all_hosts or something else?
+  hosts: OSEv3
+  roles:
+    - openshift_health_checker
+  post_tasks:
+    - action: openshift_health_check
+      args:
+        checks:
+          - disk_availability
+          - memory_availability
```

Comment:
Idea: this block might become a `playbooks/openshift-check/pre-install.yml`, and then we simply include it here.